docs/3_guides/0_submitting_proofs.md

```diff
@@ -125,12 +129,12 @@ aligned submit \
 --proving_system SP1 \
 --proof <proof_file> \
 --vm_program <vm_program_file> \
---batcher_url wss://batcher.alignedlayer.com \
+--batcher_url wss://mainnet.batcher.alignedlayer.com \
```

Comment:
Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error